script/bundled-gem-release-dates.rb

```diff
@@ -1,6 +1,12 @@
 require 'open3'
 require 'psych'
 
+# NOTE: I have tried to use change directory into a specific path
```

Comment:
might be nice to third-person-ize this text? who is "i" to the reader 5 years from now?